e2e/pages/events/enterRepresentativesEvent.page.js

```diff
@@ -65,4 +65,16 @@ module.exports = {
         throw new Error(`Unsupported representative serving preferences ${servingPreferences}`);
     }
   },
+
+  async setRepresentativeEmail(elementIndex, email) {
+    I.fillField(this.fields(elementIndex).representative.email, email);
+  },
+
+  async setDigitalServingPreferences(elementIndex) {
+    I.checkOption(this.fields(elementIndex).representative.servingPreferences.digitalService);
+  },
+
+  async setRepresentativeRole(elementIndex, role) {
+    I.selectOption(this.fields(elementIndex).representative.role, role);
+  },
```

Comment:
Should not need async, these methods are not doing anything asynchronous.

Reply:
updated the methods and removed unused methods